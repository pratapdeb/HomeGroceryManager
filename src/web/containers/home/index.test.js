import React from 'react';
import { Provider } from 'react-redux'
import Home from './index'
import configureStore  from 'redux-mock-store'
import renderer from 'react-test-renderer'

const mockStore = configureStore([])
const props = {
    getPlanets: () => {},
    getVehicles: () => {},
    findFalcon: () => {},
    clearResult:() => {},
    form: {getFieldDecorator:  jest.fn(o=>c=>c)}
}

window.matchMedia = window.matchMedia || function() {
    return {
        matches: false,
        addListener: function() {},
        removeListener: function() {}
    };
};

describe('<DestinationCard/>', () => {

    let store 
    let component

    beforeEach(() =>  {
        store  = mockStore({})
        component = renderer.create(
            <Provider store={store}>
                <Home.WrappedComponent  {...props}/>
            </Provider>
        )
    })

    it('Matches the snapshot', () => {
        expect(component.toJSON()).toMatchSnapshot()
    })
})