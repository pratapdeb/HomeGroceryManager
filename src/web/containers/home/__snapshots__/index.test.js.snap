// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DestinationCard/> Matches the snapshot 1`] = `
<section
  className="ant-layout layout"
>
  <header
    className="ant-layout-header header"
  >
    <div
      className="logo"
    />
    <img
      alt="Logo"
      src="logo.svg"
    />
  </header>
  <main
    className="ant-layout-content content"
  >
    <div
      className="ant-row"
      style={
        Object {
          "marginLeft": -8,
          "marginRight": -8,
        }
      }
    >
       
      <div
        className="ant-col ant-col-8 column"
        style={
          Object {
            "paddingLeft": 8,
            "paddingRight": 8,
          }
        }
      >
        <div
          className="ant-card ant-card-bordered"
        >
          <div
            className="ant-card-head"
            style={Object {}}
          >
            <div
              className="ant-card-head-wrapper"
            >
              <div
                className="ant-card-head-title"
              >
                Destination1
              </div>
            </div>
          </div>
          <div
            className="ant-card-body"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="ant-col ant-col-8 column"
        style={
          Object {
            "paddingLeft": 8,
            "paddingRight": 8,
          }
        }
      >
        <div
          className="ant-card ant-card-bordered"
        >
          <div
            className="ant-card-head"
            style={Object {}}
          >
            <div
              className="ant-card-head-wrapper"
            >
              <div
                className="ant-card-head-title"
              >
                Destination2
              </div>
            </div>
          </div>
          <div
            className="ant-card-body"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="ant-col ant-col-8 column"
        style={
          Object {
            "paddingLeft": 8,
            "paddingRight": 8,
          }
        }
      >
        <div
          className="ant-card ant-card-bordered"
        >
          <div
            className="ant-card-head"
            style={Object {}}
          >
            <div
              className="ant-card-head-wrapper"
            >
              <div
                className="ant-card-head-title"
              >
                Destination3
              </div>
            </div>
          </div>
          <div
            className="ant-card-body"
            style={Object {}}
          />
        </div>
      </div>
      <div
        className="ant-col ant-col-8 column"
        style={
          Object {
            "paddingLeft": 8,
            "paddingRight": 8,
          }
        }
      >
        <div
          className="ant-card ant-card-bordered"
        >
          <div
            className="ant-card-head"
            style={Object {}}
          >
            <div
              className="ant-card-head-wrapper"
            >
              <div
                className="ant-card-head-title"
              >
                Destination4
              </div>
            </div>
          </div>
          <div
            className="ant-card-body"
            style={Object {}}
          />
        </div>
      </div>
       
    </div>
    <button
      className="ant-btn ant-btn-primary button"
      onClick={[Function]}
      type="button"
    >
      <span>
         Find Felcon
      </span>
    </button>
  </main>
  <footer
    className="ant-layout-footer footer"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
     Â©2021  Finding AI Falcon
  </footer>
</section>
`;
